multicluster/config/crd/bases/multicluster.x-k8s.io_serviceexports.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: serviceexports.multicluster.x-k8s.io
+spec:
+  group: multicluster.x-k8s.io
```

Comment:
Hi @jianjuns , I am wondering if it's OK to copy serviceexports CRD definition from mcs api repo? or we should move it as own one API group?

Reply:
I feel ok to copy the yaml. Let us see if @tnqn has an opinion.

Reply:
yes, I was thinking to move it in a separate PR when we rename the `core` package.